lib/src/common.d.ts

```diff
@@ -1,16 +1,72 @@
 export declare let oDataQueryNames: string[];
 export declare const DEFAULT_VERSION = "v1.0";
 export declare const GRAPH_BASE_URL = "https://graph.microsoft.com/";
+/**
+ * @constant
+ * A package version
+ * @NOTE: This should be kept up to date with the version used in package.json.
+ * If you are changing this please ensure you are also changing it in package.json.
+ */

```

Comment:
Comment: I'd like for scenarios to be checked with either a pre-commit hook script or via CI.

Reply:
sounds like a very good option. will do that.!